dxr/indexers.py

```diff
@@ -224,6 +224,9 @@ def links(self):
 
             (sort order, heading, [(icon, title, href), ...])
 
+            File views will replace any {{line}} within the href with the
+            last-selected line number.
```

Comment:
So we scroll to the first-selected line but blame to the last. Any reason for the inconsistency?